doc/contributing/releases.md

```diff
@@ -1217,8 +1217,13 @@ the releaser, these must be kept in sync with `main`.
 The `vN.x` and `vN.x-staging` branches must be kept in sync with one another
 up until the date of the release.
 
-The TSC should be informed of any `SEMVER-MAJOR` commits that land within one
-month of the release.
+No `SEMVER-MAJOR` pull requests should be landed within one month prior to
+the major release date, unless there is consensus from the Node.js releasers
+team to do so. This measure aims to ensure better stability for the release
```

Comment:
Oh I didn't read it like that, so you mean like the following?
```suggestion
If a `SEMVER-MAJOR` pull requests lands on the default branch within one month prior to
the major release date, it should not be included on the new major staging branch, unless there is consensus from the Node.js releasers
team to do so. This measure aims to ensure better stability for the release
```

Reply:
That works. Thanks!